addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3151,9 +3151,14 @@ export class OdooEditor extends EventTarget {
         const editorTopPos = Math.max(0, editorRect.top);
         const scrollX = document.defaultView.scrollX;
         const scrollY = document.defaultView.scrollY;
+        const rangeRects = [...range.getClientRects()];
+        const rangeSpansMultipleLines =
+            rangeRects.length > 1 && rangeRects.at(-1).bottom - rangeRects[0].bottom > 2;
```

Comment:
why `> 2` instead of `> 0` ?

Reply:
Trial and error :grimacing: 
I noticed that what's rendered in the same line might have a 1, or even 2 pixels difference in the y coordinate at the bottom of the line...
I suggest you console.log the result of `rangeSpansMultipleLines` and play a tiny bit with it...

Reply:
In my tests, 0 gives too many false positives, 1 a few ones, and 2 is fine.

Reply:
Can you add a comment about it?

Reply:
Side-note: Yeey we can use `Array.prototype.at` now :-)

Reply:
See if you prefer it this way.

1st push: rebase
[2nd push](https://github.com/odoo/odoo/compare/6a7b07b15e6ce4d0436f6c53639f2bfb2bd5e45b..c9cc4d61c2f5f5dc593369ac80ea83474eda6beb): changes

Reply:
@Goaman I forgot to ping you on this one.
I have just rebased and solved conficts.